src/components/SignUpForm/index.tsx

```diff
@@ -34,19 +36,9 @@ const INITIAL_DATA = {
   expertise: '',
 };
 
-interface ISignUpForm {
-  setSnackbar: (
-    value: React.SetStateAction<{
-      open: boolean;
-      message: string;
-      severity: string;
-    }>,
-  ) => void;
-}
-
-const SignUpForm: React.FC<ISignUpForm> = ({ setSnackbar }) => {
+const SignUpForm: React.FC = () => {
   const navigate = useNavigate();
-
+  const { enqueueSnackbar } = useSnackbar();
```

Comment:
**thought(non-blocking):** I know this is a method from the lib, but maybe we should rename it, since the `enqueueSnackbar` name is not clear as `renderSnackbar` or something like this.

Reply:
after closing this conversation, please, remove `Ready for Review` and `Needs QA` and put `Ready for Merge`